sigs/sig-scalability/README.md

```diff
@@ -0,0 +1,30 @@
+# Argo SIG Scalability
+
+Argo SIG Scalability is focused on the scalability and performance of the Argo projects.
+
+The initial focus of the group is on Argo CD, working through the [the Argo CD Scalability Benchmarking proposal](https://github.com/argoproj/argo-cd/pull/12662). The time may come to extend the focus onto another project (e.g. Argo Workflows). This will be determined by the members of the SIG.
+
+The goals of this group are to create benchmarking tooling to objectively measure the scaliablity of the projects, and improving [the "scaling up" experience](https://argo-cd.readthedocs.io/en/stable/operator-manual/high_availability/#scaling-up) for end users.
+
+## Meetings
+SIG Scalability meetings are held on the second and forth Wednesday of each month. Meetings can be found in the [Argo Project public calendar](https://calendar.google.com/calendar/embed?src=argoproj%40gmail.com).
+
+The members of the group can be contacted on [the CNCF Slack](https://slack.cncf.io/), in the `#argo-sig-scalability` channel and by using [the `type:scalability` label](https://github.com/argoproj/argo-cd/pulls?q=is%3Apr+is%3Aopen+label%3Atype%3Ascalability) on PRs and issues.
+
+## Members
+Membership is open to any interested person in the Argo community.
+
+To join, create a pull request adding yourself to the list below.
+
+<!-- Alphanumeric order based on `Name` -->
+| Name                                               | Company   | Email |
+|----------------------------------------------------|-----------|-------|
+| [Andrew Lee](https://github.com/andklee)            | AWS       |       |
+| [Carlos Santana](https://github.com/csantanapr)     | AWS       |       |
+| [Dan Garfield](https://github.com/todaywasawesome)  | Codefresh |       |
+| [Jann Fischer](https://github.com/jannfis)          | Red Hat   |       |
+| [Jun Duan](https://github.com/waltforme)            | IBM       |       |
```

Comment:
Hi!
Can you please add me to this group as well?
`| [Leonardo Luz Almeida](https://github.com/leoluz) | Intuit    |       |`